lib/periodic_jobs.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+PERIODIC_JOBS = lambda { |mgr|
+  mgr.tz = ActiveSupport::TimeZone.new('America/New_York')
+  mgr.register('0 0 * * *', 'EducationForm::DeleteOldApplications')
+  # Clear out processed 22-1990 applications that are older than 1 month
+  mgr.register('20 0 * * *', 'TestUserDashboard::DailyMaintenance')
```

Comment:
This is slightly different from what's in `sidekiq_scheduler.yml (12:20am compared to 12:00. Not a big difference, so I'm not sure it matters)

Reply:
Nice. Sounds like a good idea. Do the teams that scheduled these jobs need to know about the changes? (I'm guessing it's probably not a big deal, but I'm just thinking through things.)

Reply:
Also like this approach. We should probably make a general announcement for the changes